libs/pageserver_api/src/key.rs

```diff
@@ -472,6 +527,9 @@ pub const AUX_FILES_KEY: Key = Key {
     field6: 2,
 };
 
+pub const RAW_VALUE_KEY_PREFIX: u8 = 0x04;
```

Comment:
Let's be thoughtful about the number we use here: in future, it will be a useful simplification if we can "draw a line" in the key space and say everything above it is metadata, and everything below is data.  That might motivate setting this value to something higher (e.g. 0x80, but needs more thought), so that we have space for more data prefixes in future.